Yoast/wordpress-seo
# Cleanup global overrides

## Context


* Fixes several CS issues related to global overrides. In some cases, this means prefixing variables. In others, it results in adding an ignore rule.

## Summary


This PR can be summarized in the following changelog entry:

* Fixes a variety of CS issues related to global variables being used and/or overwritten.

## Relevant technical choices:

* Fixes several CS issues related to global overrides.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Checkout this branch and build accordingly (`composer install`, `yarn` and `grunt build`).
* Navigate to the Yoast notification center. 
* Ensure that you don't see any warnings or fatal errors (also check Query Monitor).
* Navigate to SEO -> Search Appearance -> Breadcrumbs. Enable breadcrumbs.
* Ensure that you don't see any warnings or fatal errors (also check Query Monitor).
* Navigate to SEO -> Search Appearance -> Content Types.
* Ensure that you don't see any warnings or fatal errors (also check Query Monitor).
* Navigate to SEO -> Tools -> Import and Export.
* Ensure that you don't see any warnings or fatal errors (also check Query Monitor).

### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* Navigate to the Yoast notification center. 
* Ensure that you don't see any warnings or fatal errors (also check Query Monitor).
* Navigate to SEO -> Search Appearance -> Breadcrumbs. Enable breadcrumbs.
* Ensure that you don't see any warnings or fatal errors (also check Query Monitor).
* Navigate to SEO -> Search Appearance -> Content Types.
* Ensure that you don't see any warnings or fatal errors (also check Query Monitor).
* Navigate to SEO -> Tools -> Import and Export.
* Ensure that you don't see any warnings or fatal errors (also check Query Monitor).

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance
